.github/workflows/release-internal.yml

```diff
@@ -0,0 +1,64 @@
+name: Release internal patch
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: "The release version number (i.e. 1.0.0b1)"
+        type: string
+        required: true
+      sha:
+        description: "The sha to use (leave empty to use latest on main)"
+        type: string
+        required: false
+      package_test_command:
+        description: "Package test command"
+        type: string
+        default: "python -c \"import dbt.adapters.redshift\""
+        required: true
```

Comment:
Will we realistically ever want to use any other test command?  If not, drop this as an input and make it an env var.  The less inputs there can be, the easier this is to submit.

Reply:
This is something we just added to other workflows because the test command used to be `dbt --version`, which no longer works on an adapter. I don't think it's necessarily a bad thing to keep this here until we settle on what this command should be.